src/Tizen.NUI/src/public/TransitionOptions/TransitionOptions.cs

```diff
@@ -45,6 +48,23 @@ public TransitionOptions(Window window)
             mainWindow = window;
         }
 
+
+        /// <summary>
+        /// Set animated view of seamless animation.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View MainAnimatedView
```

Comment:
View and return type View is duplicated.
from next time, please use other naming. 
such as MainAnimatedTarget, MainAnimatedObject ?

Reply:
Done -> Change AnimatedTarget